types/index.d.ts

```diff
@@ -322,16 +322,21 @@ declare namespace unified {
     /**
      * Transform file contents into an AST
      *
-     * @param file File to transform into AST node(s)
+     * @param text Text to transfomr into AST node(s)
+     * @param file File associated with text
+     * @returns Parsed AST node/tree
      */
-    parse(file: VFileCompatible): Node
+    parse(text: string, file: VFile): Node
```

Comment:
Do these get passed to the parse method? looking at
https://github.com/unifiedjs/unified/blob/6c84a1f118eaed1606fa17a8d10f2bbef66d7ad3/index.js#L271
it looks like these are passed to the constructor, and that `parse` takes no args.